public/home.js

```diff
@@ -248,12 +282,28 @@ $(document).ready(function(){
 
   // delete workout post
   $(document).on('click', '.deleteBTN', function(){
-  console.log('delete post')
-  workout_id = $(this).attr('data-tag')
-  apiAction.deletePost(workout_id);
-  apiAction.getAllPost()
+    console.log('delete post')
+    workout_id = $(this).attr('data-tag')
+    apiAction.deletePost(workout_id);
+    apiAction.getAllPost();
+  
   })
   
+  // join workout
+  $(document).on('click', '#joinWorkOut', function(){
```

Comment:
perhaps more clear to find the specific element and use jQuery [click](https://api.jquery.com/click/)

```js
$('#joinWorkOut').on('click', , function(){
```